classes/Hook.php

```diff
@@ -1118,14 +1119,25 @@ private static function getAllHookRegistrations(Context $context, ?string $hookN
                 );
             }
             if (Validate::isLoadedObject($context->currency)) {
+                $currencies = [
+                    (int) $context->currency->id
+                ];
+
```

Comment:
I don't quite get why it works this way, but that's how it's treats "Customer currency" in bankwire and checkpayment modules.

Reply:
Maybe add a comment inside the code instead of githbu to explain it :sweat_smile:

Reply:
@JevgenijVisockij Always need a comment